OnlineGameStore.Infrastructure/Startup.cs

```diff
@@ -0,0 +1,31 @@
+﻿using Microsoft.Extensions.DependencyInjection;
+using OnlineGameStore.Infrastructure.Repositories.Interfaces;
+using OnlineGameStore.Infrastructure.Context;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Configuration;
+using OnlineGameStore.Infrastructure.Repositories.Implementations;
+
+namespace OnlineGameStore.Infrastructure
+{
+    public static class Startup
+    {
+        public static IServiceCollection AddPersistence(this IServiceCollection services, IConfiguration configuration)
+        {
+            return services.AddDbContext<GamesContext>((provider, options) =>
+            {
+                var connectionString = configuration.GetConnectionString("DbConnection");
```

Comment:
Move config key to constant, bind config to object

Reply:
Done, thnx